### PR TITLE
Fix Margins - Issue #108

### PR DESCRIPTION
**Pull request by tolgaouz**

### CODE REVIEW

**Comment by davidgatti** [comment]
@tolgaouz Don't forget to rework this.

File: contact/index.html [hunk 1]
```diff
@@ -5,7 +5,6 @@
 meta-description: Contact Page
 ---
   <script src="/assets/js/aws-sdk-2.656.0.min.js"></script>
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```
Comment (davidgatti) [inline contact/index.html:8]: @tolgaouz why did you move jquerry from here? It is used only in the contact page, no need to load it in each page.

Comment (tolgaouz) [reply contact/index.html:8]: I thought it wouldn't be a big issue since we can use jquery everywhere and sometimes it doesn't get loaded in the contact page itself, which I think is because loading hierarchy

Comment (davidgatti) [reply contact/index.html:8]: OK, will see.
